packages/react-native-reanimated/android/src/main/cpp/worklets/CMakeLists.txt

```diff
@@ -20,10 +20,7 @@ add_library(
 target_include_directories(
     worklets
     PUBLIC
-    "${WORKLETS_COMMON_CPP_DIR}/Registries"
-    "${WORKLETS_COMMON_CPP_DIR}/SharedItems"
-    "${WORKLETS_COMMON_CPP_DIR}/Tools"
-    "${WORKLETS_COMMON_CPP_DIR}/WorkletRuntime"
+    "${COMMON_CPP_DIR}"
```

Comment:
That's a good point. I'll check if we can do something about it now, otherwise I'll add a comment according to your suggestion.

Reply:
Actually, it looks like we can't really do much about it.

react-native also does the same (e.g. `react_render_core`): https://github.com/facebook/react-native/blob/fc8224036b4b5880c881f471e432ee5b47113b0b/packages/react-native/ReactCommon/react/renderer/core/CMakeLists.txt#L20

Reply:
Actually, this can be detected during lint phase. 

https://github.com/software-mansion/react-native-reanimated/blob/af16d243dce12ce0deb27f991af4351fad7fbb83/packages/react-native-reanimated/scripts/validate-includes.sh#L19-L21

Done in af16d243dc